application/src/js/cms/_scroll_past_sticky_status_banner.js

```diff
@@ -0,0 +1,28 @@
+;(
+    function () {
```

Comment:
If we wanted to follow the [govuk-frontend javascript style guide](https://github.com/alphagov/govuk-frontend/blob/master/docs/contributing/coding-standards/js.md) we could convert this to a named global function (eg `ErrorSummaryWithStickyBanner`?) and then do everything else within an `init()` function on the prototype?